components/src/core/Drawer/DrawerNavGroup.tsx

```diff
@@ -217,9 +214,9 @@ function NavigationListItem(
         activeIconColor = theme.palette.type === 'light'
             ? theme.palette.primary.main
             : theme.palette.primary.contrastText,
-        fontColor = theme.palette.text.secondary,
```

Comment:
Should we be making this change here, or should we be updating the secondary text color to gray[500] in the theme?

Reply:
I would prefer leaving this here. I made a branch a week ago to update that in the theme package https://github.com/pxblue/themes/tree/bug/style-adjust